<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="interface_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Interface:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interface_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gateway_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gateway:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gateway_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="target_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Target_ip:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Description_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Description_text">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="attack_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ARP Attack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scan_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scan Hosts</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
